### PR TITLE
Feat/63 implement health endpoint

### PR DESCRIPTION
**Pull request by Awambeng**
added the unit test for the health endpoint

### CODE REVIEW

**Comment by Awambeng** [comment]
@stephane-segning please review the last point under ticket [Implement health Endpoint for Health Check in PowerPay Server #63](https://github.com/ADORSYS-GIS/e2e-banking-app/issues/63)

File: power-pay-backend/pom.xml [hunk 1]
```diff
@@ -68,7 +68,6 @@
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-test</artifactId>
-            <scope>test</scope>
```
Comment (stephane-segning) [inline power-pay-backend/pom.xml:71]: Why are you removing this scope `test`?

File: power-pay-backend/src/main/java/com/adorsys/gis/powerpay/powerpaybackend/controller/HealthController.java
Comment (stephane-segning) [inline power-pay-backend/src/main/java/com/adorsys/gis/powerpay/powerpaybackend/controller/HealthController.java:1]: This is already provided by actuator. No need to implement it yourself. But good attempt!

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Good work @chendiblessing!

**Comment by chendiblessing** [comment]
@stephane-segning i have made the changes you requested

File: power-pay-backend/src/main/java/com/adorsys/gis/powerpay/powerpaybackend/controller/HealthController.java
Comment (stephane-segning) [inline power-pay-backend/src/main/java/com/adorsys/gis/powerpay/powerpaybackend/controller/HealthController.java:1]: Remove this. Actuator already provides this for you.

**Review by stephane-segning — CHANGES_REQUESTED** [review]

**Comment by Awambeng** [comment]
Prof @stephane-segning changes done

File: power-pay-backend/pom.xml [hunk 2]
```diff
@@ -27,10 +27,10 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-jpa</artifactId>
         </dependency>
-        <dependency>
+<!--         <dependency>
```
Comment (stephane-segning) [inline power-pay-backend/pom.xml:30]: What is this change?

File: power-pay-backend/src/test/java/com/adorsys/gis/powerpay/powerpaybackend/PowerPayBackendHealthTest.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+@ContextConfiguration(classes = { PowerPayBackendApplication.class })
+public class PowerPayBackendHealthTest {
```
Comment (stephane-segning) [inline power-pay-backend/src/test/java/com/adorsys/gis/powerpay/powerpaybackend/PowerPayBackendHealthTest.java:15]: This is testing the best case. You should also test the case the server is not sending 200 ok

**Review by stephane-segning — CHANGES_REQUESTED** [review]

**Comment by Awambeng** [comment]
hello prof @stephane-segning i did the last changes you demanded please you can now review

**Comment by Awambeng** [comment]
@stephane-segning everything well done

**Review by stephane-segning — APPROVED** [review]
Good work Dr @Awambeng !

**Pull request merged**